plugins/inputs/syslog/README.md

```diff
@@ -0,0 +1,110 @@
+# syslog input plugin
+
+Collects syslog messages as per RFC5425 or RFC5426.
+
+It can act as a syslog transport receiver over TLS (or TCP) - ie., RFC5425 - or over UDP - ie., RFC5426.
+
+This plugin listens for syslog messages following RFC5424 format. When received it parses them extracting metrics.
+
+### Configuration:
+
+#### TCP
+
+The minimal configuration is the following:
+
+```toml
+[[inputs.syslog]]
+  address = ":6514"
```

Comment:
Just have a single configuration, no need to show it in multiple potential configurations.

Reply:
Ok, what configuration should we show? The minimal one?

Reply:
Done showing the config for TLS.

Reply:
Run `telegraf -usage syslog` and use the output.

Reply:
Done.